app.js

```diff
@@ -44,7 +44,28 @@ if (process.env.NODE_ENV == 'production') {
   });
   
 }
-
+function logging(req, res, next) {
+  const event = {
+    referer: req.headers.referer,
+    method: req.method,
+    url: req.originalUrl,
+    body: req.body.query
+  }
+  console.log("REQUEST\n" + JSON.stringify(event, null, 2));
+  
+  res.on('finish', () => {
+    const finishEvent = {
+      statusCode: res.statusCode,
+      statusMessage: res.statusMessage
+    }
+    if (finishEvent.statusCode >= 400) {
```

Comment:
would it be better to just check if `statusCode != 200`?
I'm indifferent either way.
```suggestion
    if (finishEvent.statusCode != 200) {
```

Reply:
I had this at `>=400` , since we sometimes have redirect on our docs, but this isn't used in our docs. Gonna keep it this way, if in case we ever do want to use it for other things.